test/lib/mayaUsd/fileio/testAddMayaReference.py

```diff
@@ -135,7 +135,7 @@ def testDefineInVariant(self):
         # and that it has the expected metadata.
         attr = mayaRefPrim.GetAttribute('mayaReference')
         self.assertTrue(attr.IsValid())
-        self.assertEqual(attr.Get().resolvedPath, self.mayaSceneStr)
+        self.assertTrue(os.path.samefile(attr.Get().resolvedPath, self.mayaSceneStr))
```

Comment:
This test is failing with AssetResolver v2 because the driver letter is lower vs upper case. So the strings don't compare. Thus just added the ```os.path.samefile()``` condition.